yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -1034,27 +1034,43 @@ func fetchUnsupportedPGFeaturesFromSchemaReport(schemaAnalysisReport utils.Schem
 	}), nil
 }
 
-func getIndexesOnComplexTypeUnsupportedFeature(schemaAnalysisReport utils.SchemaReport, unsupportedIndexDatatypes []string) UnsupportedFeature {
+func getIndexesOnComplexTypeUnsupportedFeature(schemaAnalysisReport utils.SchemaReport, unsupportedIndexDatatypes []string) []UnsupportedFeature {
 	// TODO: include MinimumVersionsFixedIn
-	indexesOnComplexTypesFeature := UnsupportedFeature{
-		FeatureName: "Index on complex datatypes",
-		DisplayDDL:  false,
-		Objects:     []ObjectInfo{},
-	}
-	unsupportedIndexDatatypes = append(unsupportedIndexDatatypes, "array")             // adding it here only as we know issue form analyze will come with type
-	unsupportedIndexDatatypes = append(unsupportedIndexDatatypes, "user_defined_type") // adding it here as we UDTs will come with this type.
-	for _, unsupportedType := range unsupportedIndexDatatypes {
-		indexes := getUnsupportedFeaturesFromSchemaAnalysisReport(fmt.Sprintf("%s indexes", unsupportedType), fmt.Sprintf(queryissue.INDEX_ON_COMPLEX_DATATYPE_ISSUE_DESCRIPTION, unsupportedType), "", schemaAnalysisReport, false)
-		for _, object := range indexes.Objects {
-			formattedObject := object
-			formattedObject.ObjectName = fmt.Sprintf("%s: %s", strings.ToUpper(unsupportedType), object.ObjectName)
-			indexesOnComplexTypesFeature.Objects = append(indexesOnComplexTypesFeature.Objects, formattedObject)
-			if indexesOnComplexTypesFeature.DocsLink == "" {
-				indexesOnComplexTypesFeature.DocsLink = indexes.DocsLink
-			}
-		}
-	}
-	return indexesOnComplexTypesFeature
+	log.Infof("fetching unsupported features for Index on complex datatypes...")
```

Comment:
I believe this is just for backwards compatibility (to support yugabyted). RIght?

Reply:
yeah, and also we are appending the assessment issue in this function `getUnsupportedFeaturesFromSchemaAnalysisReport` itself, so we need to call this for all various issue types